packages/extensionmanager/src/widget.tsx

```diff
@@ -606,14 +567,14 @@ export class ExtensionView extends VDomRenderer<ListModel> {
 
       content.push(
         <CollapsibleSection
-          header={model.query ? 'Search Results' : 'Discover'}
           key="search-section"
-          startCollapsed={true}
-          onCollapse={(collapsed: boolean) => {
-            if (!collapsed && model.query === null) {
-              model.query = '';
-            }
-          }}
+          isOpen={false}
+          header={model.query ? 'Search Results' : 'Discover'}
+          // onCollapse={(collapsed: boolean) => {
```

Comment:
Looks like this should be deleted.

Not exactly sure what this original logic was for. Seems like it is setting the query to `''` if it was `null`?

Did you move this logic somewhere else?

Reply:
This is provided by blueprint's Collapse component